.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: python
 env:
-  - TWISTED=Twisted==12.2
-  - TWISTED=Twisted==12.3
-  - TWISTED=Twisted==13.0
-  - TWISTED=Twisted==13.1
```

Comment:
For 2 reasons:

1. It's not possible to use HTTPS safely with twisted <= 13.1

2. In this branch we have started using some functions that are only present starting from twisted 13.2 (namely `readBody`)

I don't think this is a big deal given the fact that debian-backports includes twisted 13.2 and maintaining backward compatibility with such old versions of twisted I expect to be increasingly painful in the long run.